baselines/bigquery-storage/package.json

```diff
@@ -24,7 +24,7 @@
     "test": "c8 mocha build/test"
   },
   "dependencies": {
-    "google-gax": "^2.2.0"
+    "google-gax": "git://github.com/googleapis/gax-nodejs.git#operation-client-type"
```

Comment:
Prolly want to block on that getting changed :)

Reply:
right, I have submitted that PR this morning and will wait for the new release to unblock this. Changing the version back now.

Reply:
done.